src/vivarium/framework/results/interface.py

```diff
@@ -23,13 +23,13 @@
     UnstratifiedObservation,
 )
 from vivarium.manager import Interface
-from vivarium.types import ScalarValue
+from vivarium.types import ScalarMapper, VectorMapper
 
 if TYPE_CHECKING:
     from vivarium.framework.results.manager import ResultsManager
 
 
-def _required_function_placeholder(*args, **kwargs) -> pd.DataFrame:
+def _required_function_placeholder(*args: Any, **kwargs: Any) -> pd.DataFrame:
```

Comment:
This is interesting! In reality, we do know the argument types here (b/c we only use this for required_updated, _formatter, and _gatherer), but since this is a sort of sentinel value it just doesn't matter.